test/agentchat/contrib/test_agent_builder.py

```diff
@@ -2,43 +2,54 @@
 import os
 import json
 import sys
-from packaging.requirements import Requirement
 from autogen.agentchat.contrib.agent_builder import AgentBuilder
-from autogen import UserProxyAgent
 from conftest import skip_openai
 
-sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
-from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
-
 here = os.path.abspath(os.path.dirname(__file__))
-oai_config_path = OAI_CONFIG_LIST
+KEY_LOC = "notebook"
+OAI_CONFIG_LIST = "OAI_CONFIG_LIST"
 
-# openai>=1 required
 try:
-    from openai import OpenAI, APIError
-    from openai.types.chat import ChatCompletion
-    from openai.types.chat.chat_completion import ChatCompletionMessage, Choice
-    from openai.types.completion import Completion
-    from openai.types.completion_usage import CompletionUsage
-    import diskcache
+    import openai
 except ImportError:
     skip = True
 else:
     skip = False or skip_openai
 
+# chromadb required
+try:
+    import chromadb
+
+    CHROMADB_INSTALLED = True
+except ImportError:
+    CHROMADB_INSTALLED = False
+
+
+def _config_check(config):
+    # check config loading
+    assert config.get("coding", None) is not None
+    assert config.get("default_llm_config", None) is not None
+    assert config.get("code_execution_config", None) is not None
+
+    for agent_config in config["agent_configs"]:
+        assert agent_config.get("name", None) is not None
+        assert agent_config.get("model", None) is not None
+        assert agent_config.get("description", None) is not None
+        assert agent_config.get("system_message", None) is not None
+
 
 @pytest.mark.skipif(
     skip,
-    reason="openai not installed OR requested to skip",
+    reason="do not run when dependency is not installed",
 )
 def test_build():
-    builder = AgentBuilder(config_path=oai_config_path, builder_model="gpt-4", agent_model="gpt-4")
+    builder = AgentBuilder(config_path=OAI_CONFIG_LIST, builder_model="gpt-4", agent_model="gpt-4")
```

Comment:
"config_path" and "OAI_CONFIG_LIST" don't seem to match semantically. The former is a path and the latter is a list.